pre_commit_hooks/xml_format.py

```diff
@@ -34,6 +36,7 @@ def main(args=None):
     if args is None:
         parser = argparse.ArgumentParser()
         parser.add_argument('filenames', nargs='*')
+        parser.add_argument('--tab-width', type=int, default=TAB_WIDTH)
```

Comment:
Hi, thanks for the contribution!

However, I'm confused, is there a commit missing here? This argparse argument you added isn't used below in the call to `fix_file`.

Reply:
Good catch! There is a missing commit indeed. I noticed the argument was missing when I was testing it locally, but forgot to push it 😅

Reply:
I figured as much, it happens all the time! I'm going to go ahead and merge this.
Thanks again for the pull request!